### PR TITLE
[Feature][docs] Document migration

### PR DESCRIPTION
**Pull request by wuchunfu**
---

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] Change does not need document change, or I will submit document change to https://github.com/InterestingLab/seatunnel-docs later

### CODE REVIEW

**Comment by leo65535** [comment]
Great job

**Comment by CalvinKirs** [comment]
hi, thank you for your contribution. Due to the excessive content, we need to check one by one, whether you can participate in the check together.

**Comment by zhongjiajie** [comment]
Did I miss somethings? I found out we just add markdown file? Does it come from https://github.com/InterestingLab/seatunnel-docs ?

**Comment by wuchunfu** [comment]
> Did I miss somethings? I found out we just add markdown file? Does it come from https://github.com/InterestingLab/seatunnel-docs ?

At present, only the configuration documents of spark and flink of 2.0 have been migrated, other documents, I will add later

**Comment by zhongjiajie** [comment]
> > Did I miss somethings? I found out we just add markdown file? Does it come from https://github.com/InterestingLab/seatunnel-docs ?
> 
> At present, only the configuration documents of spark and flink of 2.0 have been migrated, other documents, I will add later

I got it, but could we add some related issue or the plan of our doc migration to let other users/developers to get this info

**Comment by wuchunfu** [comment]
> > > Did I miss somethings? I found out we just add markdown file? Does it come from https://github.com/InterestingLab/seatunnel-docs ?
> > 
> > 
> > At present, only the configuration documents of spark and flink of 2.0 have been migrated, other documents, I will add later
> 
> I got it, but could we add some related issue or the plan of our doc migration to let other users/developers to get this info

My current plan here is
1. First migrate the configuration files of Spark and Flink
2. Check whether the migrated documents have outdated configurations and parameters
3. Waiting for the new official website to be released, other documents will be added
4. If you need to add other documents, please call me and I will do this part of the work

**Comment by zhongjiajie** [comment]
The plan sound good to me

**Comment by xtr1993** [comment]
mark  i will take

File: docs/en/flink/deployment.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+> First edit the `config/seatunnel-env.sh` in the `seatunnel` directory after decompression, and specify the required environment configuration `FLINK_HOME`
+
+## Run seatunnel on Flink Standalone cluster
+
+```bash
+bin/start-seatunnel-flink.sh \
+--config config-path
+
+# -p 2 指定flink job的并行度为2,还可以指定更多的参数，使用 flink run -h查看
```
Comment (simon824) [inline docs/en/flink/deployment.md:15]: Please translate English into Chinese.

File: docs/en/flink/deployment.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+```
+
+## Run seatunnel on Yarn cluster
+
+```bash
+bin/start-seatunnel-flink.sh \
+-m yarn-cluster \
+--config config-path
+
+# -ynm seatunnel 指定在yarn webUI显示的名称为seatunnel,还可以指定更多的参数，使用 flink run -h查看
```
Comment (simon824) [inline docs/en/flink/deployment.md:28]: ditto

File: docs/en/spark/configuration/source-plugins/Hive.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+For preprocessed `sql` , if preprocessing is not required, you can use `select * from hive_db.hive_table` .
+
+### common options [string]
+
+Source plugin common parameters, please refer to [Source Plugin](./source-plugin.md) for details
+
+**注意：使用 hive source 必须做如下配置：**
+
+```bash
+# seatunnel 配置文件中的spark section中：
```
Comment (simon824) [inline docs/en/spark/configuration/source-plugins/Hive.md:25]: ditto

**Comment by wuchunfu** [comment]
> mark i will take

If you find any problems with the documentation, please comment below.

File: docs/en/spark/configuration/source-plugins/Hive.md (hunk 3 above)
Comment (wuchunfu) [reply docs/en/spark/configuration/source-plugins/Hive.md:25]: Ok i will fix it, thx

**Comment by wuchunfu** [comment]
@CalvinKirs  I think we'd better complete the document migration work as soon as possible, because we have officially announced it to the public, I think I can do this work, please feel free to call me if you need to

**Comment by CalvinKirs** [comment]
> @CalvinKirs I think we'd better complete the document migration work as soon as possible, because we have officially announced it to the public, I think I can do this work, please feel free to call me if you need to

Is there a way to integrate the documentation with the official website? Similar to skywalking https://skywalking.apache.org/docs/main/v8.6.0/readme/

**Comment by wuchunfu** [comment]
> > @CalvinKirs I think we'd better complete the document migration work as soon as possible, because we have officially announced it to the public, I think I can do this work, please feel free to call me if you need to
> 
> Is there a way to integrate the documentation with the official website? Similar to skywalking https://skywalking.apache.org/docs/main/v8.6.0/readme/

I think I need to try it locally first.

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**